core-blocks/audio/test/index.js

```diff
@@ -3,9 +3,20 @@
  */
 import { name, settings } from '../';
 import { blockEditRender } from '../../test/helpers';
+import { registerStore } from '@wordpress/data';
 
 describe( 'core/audio', () => {
 	test( 'block edit matches snapshot', () => {
+		// The user can upload files.
```

Comment:
I don't think we really care about the state of upload buttons in terms of validating if Audio block renders properly. We only should ensure it renders properly given the default set of props. It's fine to have the edit button in the snapshot output. Can we remove this code which registers store? In my opinion, it only makes this test harder to follow.

@youknowriad mentioned that several times in different places, I just wanted to raise it again. We should seek an alternative solution for having basic validation for all blocks. Maybe e2e tests?

Reply:
The problem I tried to resolve faking a core store is that this store is undefined during the test and I get this error https://github.com/WordPress/gutenberg/pull/4155#issuecomment-397659323

Reply:
Let's skip it at the moment. I will try to tweak it myself when this lands or is about to land. It's not a blocker 👍